test.bats

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bats
```

Comment:
@neilCrosswaite, we don't want to add any more bats test, they will be removed soon anyway.